components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISProducer.java

```diff
@@ -345,7 +347,12 @@ public Document copyDocument(Exchange exchange) throws Exception {
         Folder destinationFolder = (Folder) getSessionFacade().getObjectById(destinationFolderId);
 
         Document document = (Document) getSessionFacade().getObjectById(objectId);
+        String newDocumentName = message.getHeader(PropertyIds.NAME, String.class);
+        if(!StringUtils.isEmpty(newDocumentName))
```

Comment:
Can you please use ObjectHelper instead of StringUtils?

Reply:
There is no isEmpty or notNull or some similar method in ObjectHelper class.
Am I looking at the right class ? ObjectHelper from camel-support module.

Reply:
The class from camel-util

https://github.com/apache/camel/blob/master/core/camel-util/src/main/java/org/apache/camel/util/ObjectHelper.java#L183